index.html

```diff
@@ -5,6 +5,19 @@
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+  <!-- Open Graph Tags -->
+  <meta property="og:title" content="My Pet Spotlight">
+  <meta property="og:site_name" content="mypetspotlight.com">

```

Comment:
I think `property="og:title"` will be enough, no need for `property="og:site_name"` so if you could please delete that and I'll merge the PR

Reply:
Sure

Reply:
Done